components/raftstore/src/router.rs

```diff
@@ -384,3 +384,68 @@ impl<EK: KvEngine, ER: RaftEngine> crate::coprocessor::StoreHandle for RaftRoute
         );
     }
 }
+
+/// A handle for cdc and pitr to schedule some command back to raftstore.
+pub trait CdcHandle<EK>: Clone + Send
```

Comment:
Maybe we need to find a better place for this trait. Because raftstore-v2 also implements it.

Reply:
raftstore-v2 imports raftstore, I think it's fine.